html/ajax_output.php

```diff
@@ -14,7 +14,6 @@
 
 session_start();
 if (isset($_SESSION['stage']) && $_SESSION['stage'] == 2) {
-    $_SESSION['build-ok'] = true;
```

Comment:
This was a bad error here :(  Did not allow users to restart schema build if it failed.